Rakefile

```diff
@@ -266,7 +266,7 @@ begin
             project = Xcodeproj::Project.open(project_path)
             target = project.targets.first
 
-            case target
+            case target.platform_name
```

Comment:
please add an `else` case here to raise if theres an unknown platform name, that way this can't fail silently again